components/CloseLink.js

```diff
@@ -0,0 +1,18 @@
+export default function CloseLink() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      fill="none"
+      preserveAspectRatio="xMidYMid meet"
+    >
+      <path
+        fill="#262626"
+        fillRule="evenodd"
+        d="M12 2.75a9.25 9.25 0 1 0 0 18.5 9.25 9.25 0 0 0 0-18.5ZM1.25 12C1.25 6.063 6.063 1.25 12 1.25S22.75 6.063 22.75 12 17.937 22.75 12 22.75 1.25 17.937 1.25 12Zm14.28-3.53a.75.75 0 0 1 0 1.06L13.06 12l2.47 2.47a.75.75 0 1 1-1.06 1.06L12 13.06l-2.47 2.47a.75.75 0 0 1-1.06-1.06L10.94 12 8.47 9.53a.75.75 0 0 1 1.06-1.06L12 10.94l2.47-2.47a.75.75 0 0 1 1.06 0Z"
+        clipRule="evenodd"
+      />
+    </svg>
```

Comment:
Take a look at this approach, maybe this can help you through your project:

https://codesandbox.io/s/dreamy-joana-13ij6o?file=/src/App.js